mods/seenmc.py

```diff
@@ -0,0 +1,44 @@
+import time
+import datetime
+import config
+
+from common import *
+RegisterMod(__name__)
+
+def Parse(raw, text):
+	match = re.match("^:(.[^!]+)![^@]+@[^ ]+ PRIVMSG (#[^ ]+) :(.+)", raw)
+	if match:
+		username = match.group(1)
+		channel = match.group(2)
+		message = match.group(3)
+		# TODO: settings per mod (a config file to handle what not to save here)
+		if channel != config.errorChannel and channel == "#powder-mc":
```

Comment:
Once you're using the above regex, this check won't be needed. It shouldn't look for any particular channel.

The below nick checks won't be needed either. You can have separate regexes for survival and creative if you want to differentiate server.